x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/DatafeedJobsRestIT.java

```diff
@@ -283,10 +283,12 @@ public void testLookbackOnlyWithSourceDisabled() throws Exception {
         new LookbackOnlyTestHelper("test-lookback-only-with-source-disabled", "airline-data-disabled-source").execute();
     }
 
-    @AwaitsFix(bugUrl = "This test uses painless which is not available in the integTest phase")
```

Comment:
It seems that since this test was written the test's environment is different and we have access to painless now. I re-enabled the test.